services/dummy-test-service/go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/tomdoestech/go-react-todo
+
+go 1.19
+
+require github.com/gorilla/mux v1.8.0 //direct
```

Comment:
Although gorilla does have good modules. I would advise against using a module for now. Golang's standard library already has all you need for a simple micro service as this one should be in the future.

Reply:
will remove and us standard libs

Reply:
will remove and use standard libs